load-test/bin/generate-pupil-load-test-data.js

```diff
@@ -1,101 +1,105 @@
 const winston = require('winston')
 const { TYPES } = require('tedious')
 const moment = require('moment')
-const converter = require('number-to-words')
-const R = require('ramda')
 const sqlService = require('../../admin/services/data-access/sql.service')
 const sqlPoolService = require('../../admin/services/data-access/sql.pool.service')
-const schoolDataService = require('../../admin/services/data-access/school.data.service')
-const pupilDataService = require('../../admin/services/data-access/pupil.data.service')
-const pinGenerationService = require('../../admin/services/pin-generation.service')
-const checkStartService = require('../../admin/services/check-start.service')
 
 async function main () {
   try {
-    const schoolLength = parseInt(process.argv[2])
-    if (!schoolLength) {
-      throw new Error('School length argument is not supplied or is not a valid number')
+    const pinExpireyDays = 1
+    const pinExpireyDate = moment().add(pinExpireyDays, 'day').toDate()
+
+    const numPupils = parseInt(process.argv[2])
+    if (!numPupils) {
+      throw new Error('Pupil length argument is not supplied or is not a valid number')
     }
-    // Generate additional example schools based on schoolLength
-    winston.info('Generating schools...')
-    const dfeNumbers = []
-    for (let i = 0; i < schoolLength; i++) {
-      let estabCode = 1006 + i
-      const schoolNumber = converter.toWords(6 + i)
-      const exampleSchoolNumber = schoolNumber.charAt(0).toUpperCase() + schoolNumber.slice(1)
-      estabCode = estabCode.toString()
-      let name = `Example School ${exampleSchoolNumber}`
-      const dfeNumber = 9991006 + i
-      dfeNumbers.push(dfeNumber)
-      const urn = 89006 + i
-      const params = [
-        {
-          name: 'estabCode',
-          value: estabCode,
-          type: TYPES.NVarChar
-        },
-        {
-          name: 'name',
-          value: name,
-          type: TYPES.NVarChar
-        },
-        {
-          name: 'urn',
-          value: urn,
-          type: TYPES.Int
-        },
-        {
-          name: 'dfeNumber',
-          value: dfeNumber,
-          type: TYPES.Int
-        }
-      ]
-      const sql = `INSERT INTO ${sqlService.adminSchema}.school
-      (leaCode, estabCode, name, urn, dfeNumber)
-      VALUES (999, @estabCode, @name, @urn, @dfeNumber)`
-      await sqlService.query(sql, params)
+
+    let schools, numSchools, pupilsPerSchool, pupilsRemainder, params
+
+    schools = await sqlService.query(`SELECT * FROM ${sqlService.adminSchema}.[school]`)
```

Comment:
probably best to just select the `id` field of the school, as the default dataset is 18K schools